addons/docs/src/lib/docgen/flow/createPropDef.test.ts

```diff
@@ -317,32 +317,6 @@ describe('type', () => {
     expect(type.summary).toBe('"minimum" | "maximum" | UserSize');
   });
 
-  it('removes a leading | if raw union value is used', () => {
-    const docgenInfo = createDocgenInfo({
-      flowType: {
-        name: 'union',
-        raw: '| "minimum" | "maximum" | UserSize',
-      },
-    });
-
-    const { type } = createFlowPropDef(PROP_NAME, docgenInfo);
-
-    expect(type.summary).toBe('"minimum" | "maximum" | UserSize');
-  });
-
-  it('even removes extra spaces after a leading | if raw union value is used', () => {
-    const docgenInfo = createDocgenInfo({
-      flowType: {
-        name: 'union',
-        raw: '|     "minimum" | "maximum" | UserSize',
-      },
-    });
-
-    const { type } = createFlowPropDef(PROP_NAME, docgenInfo);
-
-    expect(type.summary).toBe('"minimum" | "maximum" | UserSize');
-  });
-
```

Comment:
I don't think I removed this on purpose, might be a merge conflict that was incorrectly resolved.